.github/pull_request_template.md

```diff
@@ -6,3 +6,18 @@
 ## Solution
 
 - Describe the solution used to achieve the objective above.
+
+---
+
+## \[Optional\] Changelog
+
+- What changed as a result of this PR? If this was a trivial fix, or has no externally-visible impact, feel free to skip this section.
```

Comment:
Thoughts on something like this:

```suggestion
## Changelog

> This section is optional. If this was a trivial fix, or has no externally-visible impact, feel free to skip this section.

- What changed as a result of this PR?
```

Just to make it easier to highlight and delete all contents, so you don't have to manually remove the `[Optional]` part.

Reply:
@alice-i-cecile Sorry for the post-ready-for-final-review suggestion 😅

Reply:
> Just to make it easier to highlight and delete all contents, so you don't have to manually remove the [Optional] part.

Yeah, okay, this is a nicer workflow, and perfectly clear still.